autograph.yaml

```diff
@@ -58,35 +58,109 @@ signers:
           3utZw8En6Ik/4Om8c7EW/+EO+EkHShhgdA==
           -----END EC PRIVATE KEY-----
 
-    # p521, for testing, don't use p521
-    - id: appkey3
-      type: contentsignature
+    # Development Normandy content signer. Will issue a certificate
+    # valid for normandy.content-signature.mozilla.org. To use this
+    # with Firefox, set security.content.signature.root_hash to
+    # 5E:36:F2:14:DE:82:3F:8B:29:96:89:23:5F:03:41:AC:AF:A0:75:AF:82:CB:4C:D4:30:7C:3D:B3:43:39:2A:FE
+    #
+    # Note the private key of the root was not saved. Use tools/genpki to
+    # make a new config as needed.
+    - id: normandy
```

Comment:
This should be `normankey`

Reply:
I'm removing "normankey" entirely, the "normandy" signer replaces it.